cmd/mirror-url.go

```diff
@@ -120,12 +120,24 @@ func matchExcludeBucketOptions(excludeBuckets []string, srcSuffix string) bool {
 	} else {
 		bucketName = strings.Split(srcSuffix, "/")[0]
 	}
+	// mirror all buckets
+	if bucketName == "" {
```

Comment:
Maybe we should use * to mirror all buckets instead of an empty parameter ? it's just semantics but feels more consistent since we're using it for the pattern matching